lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -209,7 +211,33 @@ def _onInstanceableChanged(self, value):
 # Custom control for all array attribute.
 class ArrayCustomControl(object):
 
-    def __init__(self, prim, attrName, useNiceName):
+    class ArrayAEPopup(attributes.AEPopupMenu):
```

Comment:
Override this class from Maya so we can inject the copy/print menu items.

Reply:
I realized the AEPopupMenu is only available in latest Maya, so I'll need some logic here to use new vs old way.